index.html

```diff
@@ -68,25 +75,471 @@ <h3 class="main-subheading">
+          <img src="#" alt="ProductImg" class="product-img" />
+        </li>
+        <li class="product-item">
+          <img src="#" alt="ProductImg" class="product-img" />
+        </li>
+        <li class="product-item">
+          <img src="#" alt="ProductImg" class="product-img" />
+        </li>
+        <li class="product-item">
+          <img src="#" alt="ProductImg" class="product-img" />
+        </li>
+      </ul>
+    </section>
+    <section class="spotlight-container">
+      <div class="spotlight-header">
+        <h2 class="spotlight-heading">Spotlight Digital Assets Selections</h2>
+        <a href="#" class="explore-featured">Explore Featured Works &gt;</a>
+      </div>
+      <div class="spotlight-product-container">
+        <div class="spotlight-product">
+          <img src="#" alt="SpotlightProduct" />
+          <div class="spotlight-product-details">
+            <a class="spotlight-product-title">Flexsoul</a>
+            <p class="spotlight-price">$12.00</p>
+            <ul class="spotlight-tags">
+              <li class="spotlight-tag">Yoga Studio</li>
+              <li class="spotlight-tag">
+                <img src="#" alt="TagSpacer" class="spotlight-tag-spacer" />
+              </li>
+              <li class="spotlight-tag">UI Kits</li>
+            </ul>
+          </div>
+        </div>
+        <div class="spotlight-product">
+          <img src="#" alt="SpotlightProduct" />
+          <div class="spotlight-product-details">
+            <a class="spotlight-product-title">Avery</a>
+            <p class="spotlight-price">$6.00</p>
+            <ul class="spotlight-tags">
+              <li class="spotlight-tag">Monthly Report</li>
+              <li class="spotlight-tag">
+                <img src="#" alt="TagSpacer" class="spotlight-tag-spacer" />
+              </li>
+              <li class="spotlight-tag">Presentation</li>
+            </ul>
+          </div>
+        </div>
+        <div class="spotlight-product">
+          <img src="#" alt="SpotlightProduct" />
+          <div class="spotlight-product-details">
+            <a class="spotlight-product-title">Studio</a>
+            <p class="spotlight-price">$8.00</p>
+            <ul class="spotlight-tags">
+              <li class="spotlight-tag">Business Proposal</li>
+              <li class="spotlight-tag">
+                <img src="#" alt="TagSpacer" class="spotlight-tag-spacer" />
+              </li>
+              <li class="spotlight-tag">Print Template</li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="trending-container">
+      <div class="trending-header">
+        <h2 class="trending-heading">Trending Products</h2>
+        <a href="#" class="explore-trending">Explore Trending Works &gt;</a>
+      </div>
+      <div class="trending-products-container">
+        <div class="trending-product">
+          <figure class="trending-product-illustration">
```

Comment:
**Purpose:**
- The `<figure>` tag is used to encapsulate media content, such as images, diagrams, illustrations, code snippets, etc., along with an optional `<figcaption>` that provides a caption or explanation.
- The media content is essential for understanding the accompanying content.

**Usage Examples:**
- Images with captions
- Diagrams with descriptions
- Code examples with explanations
- Charts and graphs

**Characteristics:**
- The `<figure>` element is typically used to group content that is referenced in the main content but could also be moved to another location or page without losing its meaning.
- The content within a `<figure>` tag is related to the main content but is often supplementary.

here is badge is just a label and it's not a content image (we will talk about it later) 
but now you need to know, in this case, figure is enough and look like an unnecessary wrap